core/ordering/cosipbft/controller/mod_test.go

```diff
@@ -25,31 +25,65 @@ func TestMinimal_OnStart(t *testing.T) {
 	dir, err := ioutil.TempDir(os.TempDir(), "dela-test-")
```

Comment:
This is a single test function covering many cases.
Consider splitting in multiple separate test functions